internal/grpcserver/config.go

```diff
@@ -5,4 +5,5 @@ type Config struct {
 	Address                  string
 	PingIntervalInSec        int
 	PingResponseTimeoutInSec int
+    EnableReflection         bool
```

Comment:
please run `make goimports` to format the code

Reply:
@PythonHacker24 let me know once done

Reply:
Done running `make go imports` and pushed the changes